icu4c/source/common/unicode/urename.h

```diff
@@ -1053,6 +1057,7 @@
 #define ulistfmt_format U_ICU_ENTRY_POINT_RENAME(ulistfmt_format)
 #define ulistfmt_formatStringsToResult U_ICU_ENTRY_POINT_RENAME(ulistfmt_formatStringsToResult)
 #define ulistfmt_open U_ICU_ENTRY_POINT_RENAME(ulistfmt_open)
+#define ulistfmt_openForType U_ICU_ENTRY_POINT_RENAME(ulistfmt_openForType)
```

Comment:
This one is marked as `U_CAPI`, but should probably be `U_DRAFT` as well.

Reply:
Again, U_DRAFT = U_CAPI so no impl difference, but I can fix for documentation as part of API status or guard conditional update

Reply:
Thank you Peter!

Reply:
Fixed this in https://github.com/unicode-org/icu/pull/1081